### DISCUSSION TITLE
Fixing a value

### DISCUSSION THREAD

**Discussion by dhendriksen** [teslamate-org/teslamate#2988] [Q&A]
I accidentally had the per kWh charge cost for a GeoFence at $0.876 instead of $0.0876. It was only one charge, but it's off by a factor of 10 and it's going to drive me insane. I can see how to delete a record, but how can I access the database to just replace a value in a record? I know how to work with Postgres - I just don't know how to access the database.

Thank you, by the way, for building this. Awesome tool.

**Comment by cwanja** [comment]
How did you install TeslaMate? Docker? If so, check out https://docs.teslamate.org/docs/maintenance/manually_fixing_data#delete-a-drive-or-charge

The first step will show you how to drop into the database container.

**Comment by dhendriksen** [comment]
I did use Docker - running it on a Pi4. I will check that out and it does show how to delete records...I also want to insert a record. Basically I want to pull the table down, update and reload it.

**Reply by cwanja** [reply]
You mentioned you knew how to work Postgres. Once you drop into the database container, use Postgres to perform your actions. It's CLI.

**Reply by dhendriksen** [reply]
Thank you. I typically work with databases like Postgres with standard SQL commands via ODBC or JDBC. I'm not familiar with the command line but I'll do some googling.

**Reply by cwanja** [reply]
Let me try find a reply with some commands. Written a few before and shared.

**Reply by cwanja** [reply]
Check out my reply in #2699. You won't check for your drive ID, but run similar commands against `positions` table to get the IDs. Start with the last 50 or 100 rows. If you need those commands, let me know. Do not have them handy, traveling at the moment. 

As always, **backup your data**

**Reply by dhendriksen** [reply]
I got it, thank you. It just took me a second to figure out how to get to the database. I'm not used to viewing the tables in terminal and my psql syntax was a little rusty but I got it squared away. Thank you for the assist.
